tests/stub/authorization/test_authorization.py

```diff
@@ -519,10 +518,10 @@ def work(tx):
         with self.assertRaises(types.DriverError) as exc:
             session.read_transaction(work, hooks={
                 "on_send_RetryableNegative": lambda _:
-                    self.switch_unused_servers(
-                        (self._read_server1, self._read_server2),
-                        "reader_tx.script"
-                    )
+                self.switch_unused_servers(
+                    (self._read_server1, self._read_server2),
+                    "reader_tx.script"
+                )
```

Comment:
Both are not ideal. The new one I'd argue is even worse.
Usually, you want to avoid multi-line lambdas all together. Consider defining an inner function instead.

Reply:
Since this predates the PR, could we tackle this in another one? (not to mention my Python-fu is below the sea level).